ktor-client/ktor-client-darwin/darwin/src/io/ktor/client/engine/darwin/ProxySupportCommon.kt

```diff
@@ -30,3 +33,11 @@ internal fun NSURLSessionConfiguration.setupHttpProxy(url: Url) {
         HTTP_PORT_KEY to url.port
     )
 }
+
+internal fun NSURLSessionConfiguration.setupSocksProxy(url: Url) {
```

Comment:
Can be `private`? And could you also change the visibility of `setupHttpProxy`?
```suggestion
private fun NSURLSessionConfiguration.setupSocksProxy(url: Url) {
```